<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>串口调试助手</string>
  </property>
  <property name="styleSheet">
   <string>
    QMainWindow {
        background-color: #f0f0f0;
    }
    QWidget#settingsWidget {
        background-color: #ffffff;
        border: 1px solid #cccccc;
        border-radius: 5px;
        min-width: 200px;
        max-width: 250px;
    }
    QPushButton {
        padding: 5px;
        border-radius: 3px;
        background-color: #2196F3;
        color: white;
        min-height: 25px;
    }
    QPushButton:hover {
        background-color: #1976D2;
    }
    QComboBox {
        padding: 5px;
        border: 1px solid #cccccc;
        border-radius: 3px;
        background-color: white;
        min-height: 25px;
    }
    QComboBox:hover {
        border-color: #2196F3;
    }
    QLineEdit {
        padding: 5px;
        border: 1px solid #cccccc;
        border-radius: 3px;
        background-color: white;
        min-height: 25px;
    }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="settingsWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutPort">
          <item>
            <widget class="QLabel" name="labelPort">
              <property name="text">
                <string>串口号：</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QComboBox" name="comboBoxPort"/>
          </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutBaudRate">
          <item>
            <widget class="QLabel" name="labelBaudRate">
              <property name="text">
                <string>波特率：</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QComboBox" name="comboBoxBaudRate">
              <item>
                <property name="text">
                  <string>9600</string>
                </property>
              </item>
              <item>
                <property name="text">
                  <string>19200</string>
                </property>
              </item>
              <item>
                <property name="text">
                  <string>38400</string>
                </property>
              </item>
              <item>
                <property name="text">
                  <string>57600</string>
                </property>
              </item>
              <item>
                <property name="text">
                  <string>115200</string>
                </property>
              </item>
            </widget>
          </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutDataBits">
          <item>
            <widget class="QLabel" name="labelDataBits">
              <property name="text">
                <string>数据位：</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QComboBox" name="comboBoxDataBits">
              <item>
                <property name="text">
                  <string>8</string>
                </property>
              </item>
              <item>
                <property name="text">
                  <string>7</string>
                </property>
              </item>
              <item>
                <property name="text">
                  <string>6</string>
                </property>
              </item>
              <item>
                <property name="text">
                  <string>5</string>
                </property>
              </item>
            </widget>
          </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutParity">
          <item>
            <widget class="QLabel" name="labelParity">
              <property name="text">
                <string>校验位：</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QComboBox" name="comboBoxParity">
              <item>
                <property name="text">
                  <string>无校验</string>
                </property>
              </item>
              <item>
                <property name="text">
                  <string>奇校验</string>
                </property>
              </item>
              <item>
                <property name="text">
                  <string>偶校验</string>
                </property>
              </item>
            </widget>
          </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutStopBits">
          <item>
            <widget class="QLabel" name="labelStopBits">
              <property name="text">
                <string>停止位：</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QComboBox" name="comboBoxStopBits">
              <item>
                <property name="text">
                  <string>1</string>
                </property>
              </item>
              <item>
                <property name="text">
                  <string>1.5</string>
                </property>
              </item>
              <item>
                <property name="text">
                  <string>2</string>
                </property>
              </item>
            </widget>
          </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="buttonClearHistory">
         <property name="text">
          <string>清除历史</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonTogglePort">
         <property name="text">
          <string>打开串口</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonRefreshPorts">
         <property name="text">
          <string>刷新串口</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayoutRight">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QTextEdit" name="textEditHistory">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="controlLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalControlLayout">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <widget class="QPushButton" name="buttonToggleMode">
            <property name="text">
             <string>手动模式</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxChannel">
            <property name="minimumWidth">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditVoltage">
            <property name="placeholderText">
             <string>输入电压值(-9.99~9.99)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="buttonSendControl">
          <property name="text">
           <string>发送控制命令</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
</ui>
